instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/JdbcUtils.java

```diff
@@ -22,6 +23,8 @@
   private static final boolean NORMALIZATION_ENABLED = isQueryNormalizationEnabled("jdbc");
 
   private static Field c3poField = null;
+  private static final BoundedCache<String, SqlStatementInfo> sqlToStatementInfoCache =
+      BoundedCache.build(5000);
```

Comment:
a little concerned about this being pretty large, imagining 5000 hibernate queries could easily top 5MB, thoughts on dropping it to 1000?

Reply:
Oh sure.  I think @jkwatson was also thinking this number was too large, so I'll lower it to 1000.  Before I do that, I'll also sarcastically and jokingly say OMG 5 WHOLE MEGABYTES! 😝